api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,125 @@
+# Working with groups in Microsoft Graph
```

Comment:
Sorry Pat, but this is going to drive you nuts.
I want to take back what I said about "Outlook groups" and "default Office 365 groups".  While I think the latter is actually better, we've already publicly documented the former, so we should go with that (sigh).  Please see https://support.office.com/en-us/article/Learn-about-Office-365-groups-b565caa1-5c40-40ef-9915-60fdb2d97fa2 for reference.